webauthn-rs/src/lib.rs

```diff
@@ -126,6 +126,24 @@
 //! the UV *may* occur on registration and then will not occur again, and that is *by design*.
 //!
 //! If in doubt, do not enable this feature.
+//!
+//! ## Android Specific Workarounds
+//!
+//! Android has a number of issues in the dialogs they present to users for authenticator
+//! selection. Instead of allowing the user to choose what kind of passkey they want to
+//! use and create (security key, device screen unlock, sync google cloud key), Android
```

Comment:
`s/google cloud key/Google Password Manager-synchronised Passkey/`

"Google Cloud Key" may be confused with [GCP Key Management Service][0] or [Titan Security Key][1], when this is actually about [Google Password Manager][2].

[0]: https://cloud.google.com/security-key-management
[1]: https://cloud.google.com/titan-security-key
[2]: https://developers.google.com/identity/passkeys/supported-environments